.drone.star

```diff
@@ -268,15 +268,15 @@ def docker(config):
                     },
                     {
                         "name": "redis",
-                        "image": "library/redis:4.0",
+                        "image": "docker.io/redis:6",
                     },
                     {
                         "name": "email",
-                        "image": "inbucket/inbucket",
+                        "image": "docker.io/inbucket/inbucket",
                     },
                     {
                         "name": "selenium",
-                        "image": "selenium/standalone-chrome-debug:3.141.59-oxygen",
+                        "image": "docker.io/selenium/standalone-chrome:111.0",
```

Comment:
Note: `selenium/standalone-chrome-debug:3.141.59-oxygen` gets used in some core CI because of issues with the PHP-behat-mink-selenium test stack that does not support some newer webdriver stuff.

But that is not a problem here, if we are not running those web-brower tests. Let's see...

Reply:
Ref: see discussion at https://github.com/owncloud-docker/server/pull/390#discussion_r1169770353
And core issue https://github.com/owncloud/core/issues/37113